src/persistence/settings.cpp

```diff
@@ -1155,6 +1157,20 @@ void Settings::setStatusChangeNotificationEnabled(bool newValue)
     }
 }
 
+bool Settings::getShowGroupJoinLeaveMessages() const
+{
+    QMutexLocker locker{&bigLock};
+    return showGroupJoinLeaveMessages;
+}
+
+void Settings::setShowGroupJoinLeaveMessages(bool newValue)
+{
+    if (newValue != showGroupJoinLeaveMessages) {
```

Comment:
Isn't this missing a `QMutexLocker locker{&bigLock};`? You lock when getting the variable, sooo...

Reply:
Doh! Yeah, messed it up in the conflict resolution. Fixed.